packages/integration/src/ProviderLoginQuery/shared.ts

```diff
@@ -73,3 +95,19 @@ export const getMicrosoftUserInfo = async (url: string, token: string) =>
       'Content-Type': 'application/json',
     },
   }).then((response) => response.json() as Promise<Record<string, string>>);
+
+export const getJwtAndRefreshToken = (
+  id: string,
+  refreshTokenId: string,
+): { jwtToken: string; refreshToken: string } => {
+  const JWT_SECRET = getEnv('JWT_SECRET');
+  if (!JWT_SECRET) throw new Error('JWT_SECRET not set');
+
+  const JWT_SECRET_REFRESH_TOKEN = getEnv('JWT_SECRET');
+  if (!JWT_SECRET_REFRESH_TOKEN) throw new Error('JWT_SECRET_REFRESH_TOKEN not set');
+
+  return {
+    jwtToken: sign({ id }, JWT_SECRET, { expiresIn: 60 * 60 * 24 * 7 }),
```

Comment:
```suggestion
    jwtToken: sign({ id }, JWT_SECRET, { expiresIn: 60 * 60 * 24 * 7 }),
```

Can `expiresIn` be configurable? Using env for instance?